config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1910,7 +1910,7 @@ presubmits:
         - --gcp-zone=us-west1-b
         - --parallelism=1
         - --skip-regex=
-        - --focus-regex=\[Feature:CPUManager\]|\[Feature:MemoryManager\]|\[Feature:TopologyManager\]
+        - '--label-filter=Feature: isSubsetOf { CPUManager, MemoryManager, TopologyManager }'
```

Comment:
Explicitly *disabling* the default skip regexp unfortunately is necessary with kubetest2 as it stands now.